CMakeLists.txt

```diff
@@ -384,19 +383,15 @@ if(WITH_GUI)
 endif()
 
 if(WITH_WPILIB OR WITH_SIMULATION_MODULES)
-    set(HAL_DEP_REPLACE ${HAL_DEP_REPLACE_IMPL})
```

Comment:
Removing these and the IMPL versions seems like it would break installs which don't actually install the disabled bits?  That's the purpose of this, to make these replacements empty if the bits aren't actually built/installed.

Reply:
To be honest, I feel like the install has always broken for me if I disable one of the libraries. I'll revert it though.